pkg/daemon/discover/discover.go

```diff
@@ -211,16 +211,17 @@ func udevBlockMonitor(c chan string, period time.Duration) {
 		[]string{"(?i)add", "(?i)remove"},
 		udevFilter)
 
+	timeout := time.NewTimer(period)
+	defer timeout.Stop()
 	for {
-		event, ok := <-events
+		_, ok := <-events
 		if !ok {
 			return
 		}
-		timeout := time.NewTimer(period)
+		timeout.Reset(period)
 		for {
 			select {
 			case <-timeout.C:
-				break
```

Comment:
why is this `break` removed?

Reply:
The `break` statement that is needed at the end of each case in those languages is provided automatically in Go.
https://go.dev/tour/flowcontrol/9

Reply:
Thanks. Didn't know that.